### DISCUSSION TITLE
Request Phy change

### DISCUSSION THREAD

**Discussion by EdwinFairchild** [hbldh/bleak#1325] [Q&A]
It is my understanding that the server/client slave/master can all request a phy chagne. I am trying to request 2M from my Bleak based application , just not sure how?

**Comment by dlech** [comment]
Other that Android (and maybe Windows 11?) I don't think there are any operating systems APIs for this, so Bleak does not support it.

**Reply by dlech** [reply]
Not sure why #1225 was closed, but I just reopened it.

**Reply by dlech** [reply]
Although that request is for during scanning. It sounds like you are asking about during connection instead of during scanning?

**Reply by EdwinFairchild** [reply]
Correct, a better term would be phy update is guess. But yes during connection, I might work around it try to call btmgmt from within python, or maybe just have the dongle default to the PHY that I need, though a clean way to switch phys in the application would be nice.
